Entities/Timetable.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FRITeam.Swapify.Entities
+{
+    public class Timetable: BaseEntity
```

Comment:
There is a Type attribute in domain model, but I don't remember for what...

Reply:
I dont think that we need it